packages/audit/tests/test_audit_hooks.py

```
@@ -293,3 +324,68 @@ def test_keyword_match(self) -> None:
+# ---------------------------------------------------------------------------
+# main (from audit-route.py) — policy/config aliases の union 統合テスト
+# ---------------------------------------------------------------------------
+
+
+class TestMainAliasUnion:
+    """`main()` を通した alias union の統合テスト。"""
+
+    def test_policy_and_config_claude_direct_aliases_are_unioned(
+        self, tmp_path: Path, monkeypatch: pytest.MonkeyPatch
+    ) -> None:
+        """policy 側の claude-direct aliases が config 側（build_aliases）の
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)